ports/nrf/README.md

```diff
@@ -80,6 +81,7 @@ pca10000             | s110                    | Peripheral             | [Segge
 pca10001             | s110                    | Peripheral             | [Segger](#segger-targets)
 pca10028             | s110                    | Peripheral             | [Segger](#segger-targets)
 pca10031             | s110                    | Peripheral             | [Segger](#segger-targets)
+wt51822-s4at         | s110                    | Peripheral             | [Segger](#segger-targets)
```

Comment:
wt51822-s4at should be wt51822_s4at as the target name is very tied to the board folder name.

Also i wonder if we should set the "flash util" to something like "Manual debug port". As the device does not have an onboard debugger multiple debuggers could be used, but for all of them, it has to be manually connected.

Reply:
Fixed.

I've personally used (a clone of) the ST-Link v2 together with OpenOCD, which works well. Unfortunately, that means I had to flash the firmware manually (I didn't try the OpenOCD support included).